### PR TITLE
Implement build_pie

### PR DESCRIPTION
**Pull request by nrickles3**
Implemented build_pie and added sample data csv

### CODE REVIEW

File: visualize.py [hunk 1]
```diff
@@ -28,6 +52,11 @@ def main():
         tb.add_row(row)
     tb.build_table()
     tb.display()
-    
+
+
+    csv = CSVBuilder(open('data/pie_data.csv'))
```
Comment (skekre98) [inline visualize.py:57]: Let's add the open command in the __init__ function in _CSVBuilder_. WDYT?

**Comment by skekre98** [comment]
Can you also post a screenshot of what this displays when you run `../data/query.csv`

**Comment by nrickles3** [comment]
pie_chart

**Review by skekre98 — APPROVED** [review]
LGTM!

**Pull request merged**